sematic/ee/plugins/external_resource/ray/checkpoint.py

```diff
@@ -0,0 +1,44 @@
+# Sematic
+from sematic.types.registry import register_to_json_encodable_summary
+
+try:
+    # Third-party
+    import torch
+    from ray.train.torch import TorchCheckpoint  # type: ignore
+except Exception:
+    # Why "Exception" instead of just "ImportError"? Because
+    # torch can raise non-standard errors when it fails to import
+
+    # type is not available, so it just won't be registered.
+    # Note that the type is not available in "bare ray", but
+    # rather only in ray AIR.
+    TorchCheckpoint = None
+
+
+def summarize_ray_torch_checkpoint(value, _):
+    parameters = list((value.to_dict().get("model") or {}).items())
```

Comment:
```suggestion
    parameters = list((value.to_dict().get("model", {})).items())
```

Reply:
Added:
```
    # why the "or" instead of get("model", {})? Because if the dict is
    # {"model": None} we would rather replace the None with {}.
```